src/System.Net.WebSockets.Client/tests/ConnectTest.cs

```diff
@@ -29,7 +28,7 @@ public class ConnectTest : ClientWebSocketTestBase
 
                 Assert.Equal(WebSocketError.Success, ex.WebSocketErrorCode);
                 Assert.Equal(WebSocketState.Closed, cws.State);
-                Assert.Equal(ResourceHelper.GetExceptionMessage("net_webstatus_ConnectFailure"), ex.Message);
+                Assert.Equal(exceptionMessage, ex.Message);
```

Comment:
It's possible this test will fail in UAP since string resources from our CoreFx libraries are stripped out.

Instead of this `Assert.Equal`, there is a test helper "Assert" class that should be used here.

Reply:
See patterns like this in the code: ClientWebSocketUnitTest.cs:

```c#
            AssertExtensions.Throws<ObjectDisposedException>(
                () => cws.ReceiveAsync(segment, ct).GetAwaiter().GetResult(),
                expectedException.Message);
```

Reply:
Thanks for pointing this out! I vaguely remember that we need to use `AssertExtensions` for UAP run, because we don't want to validate the message, as it could be translated in future. I think this is the case.

Will fix.